frontend/packages/core/src/stories/icon-button.stories.tsx

```diff
@@ -18,7 +18,7 @@ export default {
 } as Meta;
 
 const Template = (props: IconButtonProps) => (
-  <IconButton {...props}>
+  <IconButton {...props} size="large">
```

Comment:
noticed the large size is being set in a few places - is that the new default we're going with? if so, should we update that in the iconbutton component instead?

personal preference would be to default to the medium size tho as the large looks too big to use as the default 🤔

![Screen Shot 2022-08-01 at 11 54 17 AM](https://user-images.githubusercontent.com/39421794/182190330-359f1799-0494-493e-aed0-bb4a428f7083.png)



![Screen Shot 2022-08-01 at 11 54 25 AM](https://user-images.githubusercontent.com/39421794/182190002-9419eba1-d7d4-48eb-83a9-3c354b8be38a.png)

Reply:
https://mui.com/material-ui/migration/v5-component-changes/#%E2%9C%85-update-size-prop

That's the new default by mui, which keeps the same 48px as was the default before. The only IconButtons that should've been updated were those that were imported directly from MUI vs our internal one. (So this one was a mistake :) )